### PR TITLE
rkt: Set default 'User', 'Group' to root if it's not specified.

### PR DESCRIPTION
**Pull request by yifan-gu**
cc @yujuhong @jonboulle

### CODE REVIEW

File: pkg/kubelet/rkt/rkt.go [hunk 1]
```diff
@@ -434,6 +434,17 @@ func setApp(app *appctypes.App, c *api.Container, opts *kubecontainer.RunContain
 	}
 	setSupplementaryGIDs(app, podCtx)
 
+	// If 'User' or 'Group' are still emtpy at this point,
```
Comment (jonboulle) [inline pkg/kubelet/rkt/rkt.go]: empty

Comment (yifan-gu) [inline pkg/kubelet/rkt/rkt.go]: What a catch.

**Comment by yifan-gu** [comment]
Re-apply LGTM after rebase.

**Comment by yifan-gu** [comment]
@k8s-bot test this please

**Pull request merged**